src/components/HungerAlert/HungerAlert.tsx

```diff
@@ -0,0 +1,64 @@
+'use client';
+
+import { useEffect, useState } from 'react';
+import PopupModal from '@/components/PopupModal/PopupModal';
+import DataTable from '@/components/DataTable/DataTable';
+import './style.css';
+import container from '@/container';
+import { GlobalDataRepository } from '@/domain/repositories/GlobalDataRepository';
+
+export default function HungerAlert() {
+  const [isModalOpen, setIsModalOpen] = useState(false);
+  const [countriesWithHighHunger, setCountriesWithHighHunger] = useState<
+    { rank: number; country: string; fcs: string }[]
+  >([]);
+
+  const toggleModal = () => {
+    setIsModalOpen(!isModalOpen);
+  };
+
+  useEffect(() => {
+    const fetchCountriesWithHighHunger = async () => {
+      const countryData = await container
+        .resolve<GlobalDataRepository>('GlobalDataRepository')
+        .getMapDataForCountries();
+
+      setCountriesWithHighHunger(
+        countryData.features
+          .filter(({ properties: { fcs } }) => typeof fcs === 'number' && fcs >= 0.4)
+          .sort((a, b) => (b.properties.fcs as number) - (a.properties.fcs as number))
+          .map(({ properties: { adm0_name, fcs } }, index) => ({
+            rank: index + 1,
+            country: adm0_name,
+            fcs: `${Math.floor((fcs as number) * 100)}%`,
+          }))
+      );
+    };
+
+    fetchCountriesWithHighHunger();
+  }, []);
+
+  const columns = [
```

Comment:
this should go into `operations`, you define `HungerAlertOperations` class and define a columns attribute there